pkg/network/vmispec/interface.go

```diff
@@ -102,7 +109,10 @@ func IsPodNetworkWithBridgeBindingInterface(networks []v1.Network, ifaces []v1.I
 	return true
 }
 
-func PopInterfaceByNetwork(statusIfaces []v1.VirtualMachineInstanceNetworkInterface, network *v1.Network) (*v1.VirtualMachineInstanceNetworkInterface, []v1.VirtualMachineInstanceNetworkInterface) {
+func PopInterfaceByNetwork(
+	statusIfaces []v1.VirtualMachineInstanceNetworkInterface,
+	network *v1.Network,
+) (*v1.VirtualMachineInstanceNetworkInterface, []v1.VirtualMachineInstanceNetworkInterface) {
```

Comment:
The linter fails on this line with the following error message:
> pkg/network/vmispec/interface.go:112:1: unnamedResult: consider giving a name to these results (gocritic)
func PopInterfaceByNetwork(